Fundamentals/Setup/Install/installing-nightly-builds.md

```diff
@@ -0,0 +1,101 @@
+
+Now the feed is added as a source named `Umbraco Nightly`
+
+### Using Rider
+
+To add the feed through Rider first click on the `NuGet` tab in the bottom, a menu blade will pop up, then choose the `Sources` tab. Now you must choose what NuGet configuration you wish to add the feed to, to add the feed globally add it to the `NuGet.Config` in `AppData\Roaming\NuGet`, by clicking that option. Lastly open up the `New Feed` dialog by clicking the green `+` button.
+
+![Open the new feed menu](images/Rider/open-add-feed.jpg)
+
+When you click the green `+` icon the new feed dialog will pop up, and the name and URL `https://www.myget.org/F/umbraconightly/api/v3/index.json`, to the fields, and press `OK`. Leave the User and Password empty and the `Enabled` box ticked.
+
+![Adding the feed](images/Rider/add-the-feed.jpg)
+
+Now the feed is added as a source named `Umbraco Nightly`
+
+## Finding the latest nightly version
+
+Now that the feed is added we're ready to install the nightly build, great! But this begs the question what is the version called we want to install? This is in particular an issue if we want to create a new site using the dotnet template, since the dotnet command does not allow us to use wildcards to install the newest version. Fortunately our tools can help us out here, we can see a list of available versions in both Visual Studio and Rider, and then use that version to install the template.
+
+Here we're going to assume that you want to create a brand new site with the dotnet template, however, the approach will be the same if you're updating an existing site, you'll just be clicking the update button for the `Umbraco.Cms` package instead of installing the template through the terminal.
+
+### Using Visual Studio
+
+You can use the package manager in Visual Studio to browse the available template versions. First open the package manager by selecting `Tools > NuGet Package Manager > Manage NuGet Packages For Solution...` from the top menu
+
+![Opening the Nuget Package Manager](images/VS/open-nuget-package-manager.jpg)
+
+Now the `NuGet` window will open, in the top right corner there is a dropdown `Package source`, in order to be able to see our nightly builds, we have to change this to the feed we've just added `Umbraco Nightly` in this case.
+
+![Select the nightly NuGet feed](images/VS/select-nuget-feed.jpg)
+
+Now that the correct feed is selected we're almost ready to search for the package and choose a version, however since these are marked as prerelease, we have to tick the checkbox `Include prerelase` otherwise these will be filtered out. Next search for `Umbraco.Templates`, and choose that package. Now we can click on the `Version` drop down and see the available nightly builds for both V9 and V10. It's important to note that this is the ***template***, not the Umbraco package itself, so it won't work to click install. Instead chose the applicable version and note down the version number
```

Comment:
```suggestion
Now that the correct feed is selected we're almost ready to search for the package and choose a version, however since these are marked as prerelease, we have to tick the checkbox `Include prerelease otherwise these will be filtered out. Next search for `Umbraco.Templates`, and choose that package. Now we can click on the `Version` drop down and see the available nightly builds for both V9 and V10. It's important to note that this is the ***template***, not the Umbraco package itself, so it won't work to click install. Instead chose the applicable version and note down the version number
```